src/renderer/components/dock/editor-panel.tsx

```diff
@@ -48,6 +57,24 @@ export class EditorPanel extends React.Component<Props> {
   constructor(props: Props) {
     super(props);
     makeObservable(this);
+
+    setDiagnosticsOptions({
+      validate: true,
+      enableSchemaRequest: true,
+      hover: true,
+      completion: true,
+      isKubernetes: true,
+      schemas: [
+        {
+          // Id of the first schema
+          uri: 'https://raw.githubusercontent.com/garethr/kubernetes-json-schema/master/v1.9.9/all.json',
```

Comment:
https://github.com/yannh/kubernetes-json-schema/ has more up-to-date schemas. We could pick schema that matches with current k8s cluster.

Or alternatively we could generate schema from k8s api openapi schema on-the-fly. This way it would also contain all CRDs (might make senseto implement in a separate PR).

Reply:
I think it is not a critical issue for now, I've used the link just to check how it works. Unfortunately, I can't make it work in our current environment (webpack and electron-builder). 
![image](https://user-images.githubusercontent.com/53330707/132454232-85a58036-44b8-49e3-be54-c95df6e13041.png) 
The error appears on the editor initialization, I've investigated all documentation and a huge number of stackoverflow threads, but it didn't help.